ncm-grub/src/main/perl/grub.pm

```diff
@@ -74,6 +77,66 @@ sub grubbyArgsOptions {
 
 }
 
+sub Configure_password {
+    my ($self, $config, $editor) = @_;
+
+    my $password = undef;
+    my $passwordpath = $self->prefix . "/password";
+
+    # if passwords have not been explicitly enabled or disabled then do nothing.
+    return unless ($config->elementExists($passwordpath));
+    my $tree = $config->getElement($passwordpath)->getTree();
+
+    if (!defined($tree->{enabled})) {
+        return;
```

Comment:
add a verbose message that password section was defined, but neither enabled nor disabled